### PR TITLE
Convert old style 'on'/''off' to real boolean arguments on remaining plugins

### LINKED ISSUE #3440: BP001: Fix boolean command line options
Related to #3427

### PR DESCRIPTION
**Pull request by beraldoleal**
This is removing old style 'on'/'off' from some configuration namespaces and replacing by a true bool. This is part of BP001.
Fixes #3440

### CODE REVIEW

**Review by willianrampazzo — APPROVED** [review]

File: optional_plugins/html/avocado_result_html/__init__.py [hunk 1]
```diff
@@ -309,9 +304,8 @@ def configure(self, parser):
 
         settings.add_argparser_to_option(
             namespace='job.run.result.html.enabled',
-            choices=('on', 'off'),
             parser=run_subcommand_parser,
-            long_arg='--html-job-result')
+            long_arg='--disable-html-job-result')
```
Comment (richtja) [inline optional_plugins/html/avocado_result_html/__init__.py:308]: I find this one mentioned in [documentation](https://avocado-framework.readthedocs.io/en/83.0/plugins/optional/results.html?highlight=%E2%80%93html-job-result#html-results-plugin) can you please also update  the document.

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @beraldoleal LGTM, but there is a small change in documentation that needs to be done.

File: optional_plugins/html/avocado_result_html/__init__.py (hunk 1 above)
Comment (beraldoleal) [reply optional_plugins/html/avocado_result_html/__init__.py:308]: Nice catch @richtja.

**Comment by beraldoleal** [comment]
@richtja fixed with a force-push.

**Review by richtja — APPROVED** [review]

**Pull request merged**